formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -585,3 +585,16 @@ notReallyJSX foo::1 [] bar::2 [@JSX] [@bla];
 <Optional1 required=(Some "hi") /> [@bla];
 
 <Optional1 required=(Some "hi") /> [@bla];
+
+/* Overeager JSX punning #1099 */
```

Comment:
My mistake, accidentally branched from my jsx-punning-fix-branch, that commit has been merged into master already.